upload-chunked-file.js

```diff
@@ -0,0 +1,43 @@
+function chunkFileUpload(file, serverUrl, chunkSize) {
```

Comment:
It could be beneficial to validate the `file`, `serverUrl`, and `chunkSize` inputs at the beginning of the function to ensure they are valid and prevent runtime errors.